sig-scheduling/2020-annual-report.md

```diff
@@ -0,0 +1,125 @@
+
+## Membership
+
+**Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?**
+
+Yes, except for inactive subprojects.
+
+**How do you measure membership? By mailing list members, OWNERs, or something else?**
+
+We don't have an official way of measuring membership, there is some churn in the number of active 
+members and so membership is hard to keep track of.
+
+**How does the group measure reviewer and approver bandwidth? Do you need help in any area now?
+ What are you doing about it?**
+
+PRs are usually directed to the reviewer most familiar with the code base the PR is modifying.
+We don't quite measure bandwidth, but one way of doing that is by looking at pending PRs broken
+by assignment.
+
+**Is there a healthy onboarding and growth path for contributors in your SIG? What are some 
+activities that the group does to encourage this? What programs are you participating in to 
+grow contributors throughout the contributor ladder?**
+
+There is no official onboarding process. One thing we try to do frequently is breaking up 
+larger features into smaller enough tasks for new members to contribute.
+
+**What programs do you participate in for new contributors?**
+
+Currently we don't.
+
+**Does the group have contributors from multiple companies/affiliations? Can end users/companies 
+contribute in some way that they currently are not?**
+
+Yes.
+
+## Current initiatives and project health
+
+**What are initiatives that should be highlighted, lauded, shout out, that your group is proud of?
+ Currently underway? What are some of the longer tail projects that your group is working on?**
+
+- Initiatives:
+    - Focusing on turning the scheduler into a pluggable framework to allow developing new custom
+     features outside the main repo.
+     [k-sigs/scheduler-plugins] initiated this to build a scheduler plugin ecosystem. It has a wide
+     participation from different companies, by either direct contributions from Alibaba, Tencent,
+     Apple, etc., as well as adoption by companies like [OpenAI].
+    - Improving scheduler performance. We're working on some items like optimizing internal queues,
+     tailored preemption logic, as well as exposing meaningful metrics to help define SLA/SLOs.
+- Longer tail projects:
+    - Continuing to refactor the core code around the scheduling framework.
+    - Graduating the scheduler's ComponentConfig to GA.
+
+**Year to date KEP work review: What's now stable? Beta? Alpha? Road to alpha?**
+
+- Alpha
+    - [Prefer Nominated Node]
+    - [Node Resource Strategy]
+    - [Pod Affinity Namespace Selector]
+    - [Volume Capacity Priority] (co-owned by sig-storage)
+- Beta
+    - [Default topology spread] (will graduate with CC)
+    - [Multi Scheduling Profiles]
+    - [Non-preempting priority class]
+    - [Component Config]
+
+**What areas and/or subprojects does the group need the most help with?**
+
+- **Docs improvement**:
```

Comment:
ty for calling this out

Reply:
agreed, thanks. I noticed https://docs.google.com/document/d/1t2QoASmg_BssFczl8n7UNTDWKgncMSrLTX802fQ0lK8/edit# and https://github.com/kubernetes/community/pull/5674 recently, which seemed to address some of the developer-facing bits. Would it make sense to open help-wanted issues for remaining dev doc gaps and for specific user-oriented gaps?

Reply:
Yes, @ingvagabund 's  recent contributions really make the docs greatly improved. With those docs in place, I think we're fine on dev docs. In terms of user-oriented doc gaps, we will open help-wanted issues for tracking.